AdobeStockImageAdminUi/Test/Mftf/Section/AdobeStockImagePreviewSection.xml

```diff
@@ -8,7 +8,9 @@
 
 <sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
     <section name="AdobeStockImagePreviewSection">
-        <element name="save" type="block" selector="//button[@class='action-secondary']/span[contains(.,'Save Preview')]"/>
+        <element name="save" type="block" selector="//button[@class='action-secondary']//span[text()='Save Preview']"/>
+        <element name="locateImage" type="block" selector="//button[@class='action-secondary']//span[text()='Locate')]"/>
+        <element name="saveLicensedImage" type="button" selector="//div[@class='actions']/descendant::span[text()='Save']"/>
```

Comment:
Hi @filmaj, please refer: https://github.com/magento/adobe-stock-integration/pull/639#discussion_r346739377 as it is the same issue and fix.